.changeset/hip-tables-check.md

```diff
@@ -0,0 +1,13 @@
+---
+"@latticexyz/store": major
+"@latticexyz/world": patch
```

Comment:
this got dismissed with applying the change but I am wondering about moving this to its own changeset

otherwise, the world changelog will have info about store internals changes (which aren't super relevant if you're using world)

Reply:
Since `StoreRead`'s API was changed, also the `World` API changed, so i'd say this is a `major` change for world?

Reply:
Ohh right, very true